recipe/meta.yaml

```
@@ -99,7 +101,9 @@ test:
     - commands_to_test.bat           # [win]
     - commands_to_test.sh            # [unix]
   downstreams:
-    - conda-smithy  # [python_impl == "cpython" and py<311]
+    {% if ((py>=311 and linux) or (py==pymin and win)) and python_impl == "cpython" %}
+    - conda-smithy
+    {% endif %}
```

I disagree with `py==pymin and win`. We should be testing downstream on every supported python version and across all platforms - the fact that smithy pins `python_min` only on windows is not consistent.

In any case, we need to drop the `python_impl` condition, because pypy is gone and 3.13 has a different string.

---

Using https://github.com/conda-forge/conda-smithy-feedstock/pull/331 (updated), we could still do the following:
```suggestion
    - conda-smithy  # [py>311]
```

---

It is also not consistent to not run some of the smithy tests as a downstream package.

---

> It is also not consistent to not run some of the smithy tests as a downstream package.

I'm not sure what you're saying here. I'm not proposing to reduce testing, but to increase it (or rather, keep it as before).